.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,61 @@
+landing_getting_started_1: |-
+  // In app.component.ts
+
+  import { Component } from '@angular/core'
+  import { instantMeiliSearch } from '@meilisearch/instant-meilisearch'
+
+  const searchClient = instantMeiliSearch(
+    "http://localhost:7700",
+    "searchKey"
+  )
+
+  @Component({
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.css'],
+  })
+  export class AppComponent {
+    title = 'angular-app'
+    config = {
+      indexName: 'movies',
+      searchClient,
+    }
+  }
+
+  // In app.component.ts
```

Comment:
The same as the first comment?  `// In app.component.ts` so this entire example is just one file?